source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -0,0 +1,109 @@
+# Mattermost Cloud Changelog
+
+This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
+
+## Release 2020-12-09
+
+### Improvements
+
+#### Cloud Services
+ - The Customer Web Server can now make HTTP requests to an installation by passing the X-Cloud-Token header, with the value being that installation's Customer API Key.
+ - System Admins can receive an email notifying them of a failed payment in Mattermost cloud.
+ - In the event a subscription has a payment and there is no card on the account when that payment is processed, an email will be sent to attempt to get the user to pay up.
```

Comment:
Not sure if these updates are needed in the changelog we publish. They're mainly just internal details end users/customers won't care about. If we do want to keep them I'd put them at the bottom as more of a footnote